themes/upbound/assets/scss/_home.scss

```diff
@@ -76,7 +76,7 @@
         .about-link {
             display: inline;
             a {
-                color: #6553C0 !important;
+                color: #7D4CD1 !important;
```

Comment:
Nitpick: could you use the variable `$up-purple-400` here instead? just to keep things consistent and easier to update down the link